### PR TITLE
[MAINTENANCE] Added format unit tests.

### LINKED ISSUE #781: [FUND] Add tests and significantly improve test coverage
## Description

Currently Kitodo.Presentation barely has any (unit) tests, so implementing new features and making sure that nothing broke is very tedious and time consuming - not only for developers but for the release management as well. This proposal suggests to write a lot of tests and significantly improve test coverage for Kitodo.Presentation.

## Related Issues

None.

## Expected Benefits of this Development

Test-driven development ensures that developing new features, updating components or re-factoring code doesn't break anything (or at least makes obvious what broke). This is especially important for older projects with a large and complex code base, since it's impossible for a single developer to know every constraint and dependency.
Having tests would therefor improve code quality, make development easier and help testing and reviewing pull requests for the release management.

## Estimated Costs and Complexity

Writing tests is not a complex, but a rather time-consuming task, so I'd estimate this to be of **high** cost and **low** complexity.

### PR DESCRIPTION
**Pull request by haogatyp**
Some of the tests fail because the existing code being tested appears to be buggy.

### CODE REVIEW

**Comment by sebastian-meyer** [comment]
Could you please bring this pull request up to date with master? Alternatively you can allow me to update your branch, then I can do it myself via GitHub most of the time.

**Comment by haogatyp** [comment]
> Could you please bring this pull request up to date with master? Alternatively you can allow me to update your branch, then I can do it myself via GitHub most of the time.

I did a rebase onto current master.

**Comment by sebastian-meyer** [comment]
@haogatyp Please bring your branch up-to-date with the master branch.

**Comment by oliver-stoehr** [comment]
@sebastian-meyer the pull request is fixed and updated

**Comment by sebastian-meyer** [comment]
Please have a look at the issues reported by Codacy: https://github.com/kitodo/kitodo-presentation/runs/18049153004

**Comment by sebastian-meyer** [comment]
Thank you, but there is still one last issue remaining... ;o)

File: Tests/Fixtures/Format/modsOriginInfoWithEditionElectonicEd.xml [hunk 1]
```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sebastian-meyer) [inline Tests/Fixtures/Format/modsOriginInfoWithEditionElectonicEd.xml]: There is a typo in the file name.

**Review by sebastian-meyer — CHANGES_REQUESTED** [review]
Please fix the filename (see comment) and add the default license header to every PHP file. Just copy and paste from any other code file!

After that we are finally ready for merging!

**Review by sebastian-meyer — APPROVED** [review]
Thank you so much!

**Pull request merged**